docs/installation/install-node.rst

```diff
@@ -44,6 +44,54 @@ If you do not have any nodes, the following guides will help you set some up, wi
 
 .. XXX In the integration specific documentation, links to the guides appear here
 
+.. begin-body-installing-node-rhel
+
+Installing on RHEL 7
+====================
+
+.. note:: You should ensure your nodes are Flocker-ready, either by checking the prerequisites above, or by following our guides on using Amazon Web Services or Rackspace.
+
+#. **Log into the first node as root:**
+
+   .. prompt:: bash alice@mercury:~$
+
+      ssh root@<your-first-node>
+
+#. **Install the** ``clusterhq-flocker-node`` **package:**
+
+   To install ``clusterhq-flocker-node`` on RHEL 7 you must install the RPM package provided by the ClusterHQ repository.
+   The commands below will install the two repositories and the ``clusterhq-flocker-node`` package.
+   
+   Run the following commands as root on the target node:
+
+   .. prompt:: bash root@rhel:~$
+
+      yum list installed clusterhq-release || yum install -y https://clusterhq-archive.s3.amazonaws.com/centos/clusterhq-release$(rpm -E %dist).centos.noarch.rpm
+
+      sed -i 's/$releasever/7/g' /etc/yum.repos.d/clusterhq.repo
+      yum-config-manager --enable clusterhq
+      yum install -y clusterhq-flocker-node
+
+#. **Install the** ``clusterhq-flocker-docker-plugin`` **package:**
+
+   At this point you can choose to install the Flocker plugin for Docker.
+   Run the following command as root on the target node:
+
+   .. prompt:: bash [root@rhel]#
+   
+      yum install -y clusterhq-flocker-docker-plugin
+
+   .. XXX FLOC-3454 to create a task directive for installing the plugin
```

Comment:
I added indents here (and below where the same comment was used), as it was causing the numbering to get misaligned.